tooling/templatize/cmd/pipeline/run/cmd.go

```diff
@@ -21,7 +21,7 @@ type Version struct {
 var versionConstraints = []Version{
 	{
 		Name:       "Azure CLI",
-		Cmd:        "az --version | grep azure-cli |awk '{print $NF}'",
+		Cmd:        "az  --version 2>/dev/null | grep azure-cli |awk '{print $2}'",
```

Comment:
is this in here on purpose? seems not related to theme of the PR

Reply:
It failed the PR cause output changed with new update

Reply:
ack